src/sentry/api/endpoints/organization_member/team_details.py

```diff
@@ -98,6 +124,12 @@ def _can_delete(
         if request.user.id == member.user_id:
             return True
 
+        # There is an edge case where org owners/managers cannot remove a member from a team they
+        # are not part of using team:write. We cannot explicitly check for team:write b/c org admins
+        # have it but are only allowed to remove members from teams they are on.
```

Comment:
i thought org owners & managers can remove a member from a team they aren't a part of? do you mean if they try to do this using an org token which only has `team:write`?

Reply:
Previously if they tried to do remove people from a team they weren't a part of using a user token that had `team:write`,  it would fail. This PR fixes that